deploy/templates/nstemplatetiers/appstudio/ns_tenant.yaml

```diff
@@ -102,6 +102,36 @@ objects:
       requests.storage: 50Gi
       requests.ephemeral-storage: 50Gi
       count/persistentvolumeclaims: "12"
+- apiVersion: v1
```

Comment:
I would split this one big quota to multiple quotas. The RQ.status has a filed which is updated when the actual quota usage is changed: `used`. So every time some of the resources in the quota is created the corresponding RQ is updated. If we have too many items in the quota it will lead to too many conflicts. Basically there might be a high rate of errors when a CR from this quota is created.
It can be mitigated by creating multiple quotas. Let's say one quota for SPI resources, one for HAS, one for release, etc.

Reply:
Thanks @alexeykazakov we created a follow-up PR #786. Please let us know your thoughts on it